libsrc/apple2/get_tv.s

```diff
@@ -0,0 +1,187 @@
+;
+; Colin Leroy-Mira <colin@colino.net>, 2025
+;
+; unsigned char __fastcall__ get_tv(void)
+;
+        .export         _get_tv
+
+        .import         _set_iigs_speed, _get_iigs_speed
+        .import         ostype
+
+        .constructor    calibrate_tv, 2
+
+        .include    "accelerator.inc"
+        .include    "apple2.inc"
+        .include    "get_tv.inc"
+
+; The only thing remaining from that code after init
+.proc _get_tv
```

Comment:
Just a matter of style: So far my style was to either have "just some code" without any segment assignment in the source file or to have a more complex with (several) segment assignment. The latter case I explicitly wrote .code and didn't rely on it being the default. Additionally I'm someone how puts the main() at then end of a C source. In the same way I'd put this "final ourward facing result" code at the end of this source file. And then you need .code anyhow. But as I wrote, just a question of style...